pipelines/bedbuncher.py

```diff
@@ -17,40 +17,47 @@
 from argparse import ArgumentParser
 from bbconf.const import *
 from bbconf.exceptions import BedBaseConfError
-from elasticsearch.exceptions import ConflictError
+from bbconf import BedBaseConf
+from hashlib import md5
+from psycopg2.extras import Json
 
 import bbconf
 import pypiper
 import yacman
 
-parser = ArgumentParser(description="A pipeline to produce sets of bed files (bedsets) from bedbase")
-
-parser.add_argument("-q", "--JSON-query-path", help="path to JSON file containing the query", type=str) # path to JSON with query
-parser.add_argument("-b", "--bedbase-config", type=str, required=False, default=None,
-                    help="path to the bedbase configuration file")
-parser.add_argument("-n", "--bedset-name", help="name assigned to queried bedset", type=str)
+parser = ArgumentParser(
+    description="A pipeline to produce sets of bed files (bedsets)")
+
+parser.add_argument("-q", "--query",
+                    help="condition string to restrict bedfiles table select with",
+                    type=str)
+parser.add_argument("-v", "--query-val",
+                    help="condition values to populate condition with",
+                    type=str)
+parser.add_argument("-b", "--bedbase-config",
+                    type=str, required=False, default=None,
+                    help=f"path to the bedbase configuration file. "
+                         f"Optional if {CFG_ENV_VARS} env vars are set.")
+parser.add_argument("-n", "--bedset-name",
+                    help="name assigned to the bedset to be created", type=str)
 
 # add pypiper args to make pipeline looper compatible
-parser = pypiper.add_pypiper_args(parser, groups=["pypiper"], required=["--JSON-query-path", "--bedset_name"])
-
-
+parser = pypiper.add_pypiper_args(parser,
+                                  groups=["pypiper"],
+                                  required=["--query", "--bedset_name"])
```

Comment:
1. `--bedset-name`
2. `--bedbase-config` is set as required at the definition stage and these are set as required here, let's make this consistent